### DISCUSSION TITLE
Is Vercel's documentation hosted on GitHub?

### DISCUSSION THREAD

**Discussion by hirehamir** [vercel/community#3843] [Feature Question] [answered]
If it is, could someone point me to the correct repository?

I saw `0 results` for `doc` at https://github.com/vercel?q=doc&type=all&language=&sort=.

![image](https://github.com/vercel/community/assets/43425812/1aab6181-9e72-40b2-a32f-522bbc8decb5)

If it isn't, what are your thoughts on making them open source? I think it'd be easier for readers to contribute fixes if they were.

**Comment by leerob** [comment, marked as answer]
Hey! Our docs are closed source and the process is working well for us right now. If you do see something wrong, please let us know and we'll be happy to take a look. Either through the feedback button on the specific page, or you can use Discussions as well 🙏

**Reply by hirehamir** [reply]
Okay, I'll keep that in mind. Thanks for the quick response!
